posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "2024 mpox outbreak: common analytics tasks and available R tools"
+author:
+  - name: "James Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-147"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-01-04"
+categories: [R, Epiverse-TRACE, mpox, outbreak, outbreak-analytics, DOI]
```

Comment:
```suggestion
categories: [Epiverse-TRACE, mpox, outbreak, outbreak-analytics, DOI]
```

This is not a good fit for R-bloggers based on their guidelines. Once this is committed, I can merge the post today.